src/Symfony/Component/HttpClient/Tests/CurlHttpClientTest.php

```diff
@@ -50,24 +50,19 @@ public function log($level, $message, array $context = [])
         $client = new CurlHttpClient();
```

Comment:
shouldn't this test be updated to opt-in for push, now that it is disabled by default ?

Reply:
sure, fixed
note that the test doesn't work, but we skip it
I prefer keeping it this way so that we are reminded about it in the future :)